torchbenchmark/util/kernels/triton_fused_attention.py

```diff
@@ -182,6 +183,164 @@ def _attn_fwd(Q, K, V, sm_scale, M, Out,  #
     tl.store(O_block_ptr, acc.to(Out.type.element_ty))
 
 
+@triton.jit
+def _attn_fwd_inner_tma(acc, l_i, m_i, q,  #
+                    K_desc_ptr, V_desc_ptr, Q, qvk_offset, stride_kn, stride_vn, stride_vk,  #
+                    start_m, qk_scale,  #
+                    BLOCK_M: tl.constexpr, HEAD_DIM: tl.constexpr, BLOCK_N: tl.constexpr,  #
+                    STAGE: tl.constexpr, offs_m: tl.constexpr, offs_n: tl.constexpr,  #
+                    N_CTX: tl.constexpr, fp8_v: tl.constexpr):
+    # range of values handled by this stage
```

Comment:
Nit: can you leave a comment so folks know that the required TMA fence is in `_attn_fwd_tma()`?